tests/docker/Dockerfile-radius

```diff
@@ -1,5 +1,6 @@
 FROM fedora:21
 
+RUN yum install -y yum-plugin-ovl
```

Comment:
what is the problem this change is solving?

Reply:
This fixes the issue - https://github.com/radcli/radcli/issues/46 

I face the above issue each time. If the fix is not correct, kindly assist on the same.

Reply:
I am a bit hesitant here because I cannot really reproduce it and the ovl plugin doesn't really make sense to me. Have you tried updating to a different fedora release (e.g. 27 or 28) fixes the issue? If yes, we can update that part instead. Overall I was thinking that maybe we get rid of docker dependency here and use namespaces directly with a script such as:
https://gitlab.com/openconnect/ocserv/blob/master/tests/ns.sh but never got into that. Anyway for a quick fix most likely changing the fedora release would work.

Reply:
Ok

Reply:
Hi @nmav 

I will check and get back.